backend/src/main/java/com/bakdata/conquery/models/common/daterange/CDateRange.java

```diff
@@ -320,6 +320,11 @@ public boolean hasLowerBound() {
 	 * @return The years as date ranges, from the first date in range to the last in ascending order.
 	 */
 	public List<CDateRange> getCoveredYears() {
+		if(isOpen()){
+			// TODO: 22.04.2020 throw exceptiopn?
```

Comment:
Hast du das mit Lennart abgesprochen?, ich weiß nicht wann die OpenRanges auftreten aber potentiell können dadurch dann Daten in den Formularen fehlen.

Reply:
das habe ich mit anne so besprochen, sie benutzt dafür momentan dummy variablen

Reply:
wenn wir es nicht testen ist das eine endlosschleife.

Reply:
Ok, ich habe gerade auch nochmal nachgeschaut in dem Context, sollten eigentlich auch keine offnen Ranges auf treten